components/ui/NFTGrid/index.tsx

```diff
@@ -71,7 +83,7 @@ export function NFTGrid({ data }: { data: NFT[] }) {
                     )
                 })}
             </Grid>
-            <NFTModal selectedNFT={selectedNFT} isOpen={isOpen} onClose={onClose} mintingError={mintingError} isLoading={isLoading} mintNFT={mintNFT} />
+            <NFTModal selectedNFT={selectedNFT} isOpen={isOpen} onClose={onClose} success={success} tokenId={tokenId} mintingError={mintingError} isLoading={isLoading} mintNFT={mintNFT} />
```

Comment:
would be great if you can enable/configure prettier formatting

Reply:
sure thing